rewrite-core/src/main/java/org/openrewrite/internal/NameCaseConvention.java

```diff
@@ -149,11 +149,13 @@ private static String lowerUnderscore(String str) {
     private static String toCamelCase(String str, boolean lowerCaseFirstLetter) {
         boolean allUpperCase = true;
         for (char c : str.toCharArray()) {
```

Comment:
It would be even better if we would here use a regular `for` loop and access the characters using `String#charAt(int)` rather then converting to `char[]` first.

Reply:
Sure, I'll fix that as well